core/core.go

```diff
@@ -114,9 +114,15 @@ func resetDirectories() {
 	log.Trace("Resetting file directories to a clean slate.")
 
 	// Wipe the public, web-accessible hls data directory
-	os.RemoveAll(config.PublicHLSStoragePath)
-	os.RemoveAll(config.PrivateHLSStoragePath)
-	err := os.MkdirAll(config.PublicHLSStoragePath, 0777)
+	err := os.RemoveAll(config.PublicHLSStoragePath)
+	if err != nil {
+		log.Fatalln(err)
+	}
```

Comment:
Just to save a line (and I'm not always good at remembering to do this myself), you can write these error checks as

```
if err := os.RemoveAll(config.PublicHLSStoragePath); err != nil {
   ...
}
```

Reply:
Thanks for pointing it out. I think this should be refactored once in the whole codebase to prevent people from copying the style in their contribution.